package.json

```diff
@@ -35,6 +37,7 @@
   "scripts": {
     "test": "nyc mocha --recursive --exit",
     "posttest": "eslint ./ && license-checker --summary",
-    "beautify": "eslint ./ --fix"
+    "beautify": "eslint ./ --fix",
+    "semantic-release": "semantic-release"
```

Comment:
How did you test?

Reply:
yeah that is the issue...I can't test unless I create a dummy repo (hence why I am using this not very important library to test lol)